### DISCUSSION TITLE
OpenTelemetry .Net Logging

### DISCUSSION THREAD

**Discussion by panthamm** [open-telemetry/opentelemetry-dotnet#2778] [Q&A] [answered]
I am tracing .Net application using ZipkinExporter successfully. Do we have any exporters available for Logging in .Net application?
Right now I am seeing only AddConsoleExporter for logging.

**Comment by SergeyKanzhelev** [comment]
Logging is currently in alpha stage in OpenTelemetry. So we do not have much support. But I know @cijothomas is looking into it now. @cijothomas  can you help?

**Reply by cijothomas** [reply]
As of now, we don't have any exporter other than Console. 
Please open an issue in the Otel .NET repo if you have additional .NET specific questions.

**Reply by panthamm** [reply]
Thanks @SergeyKanzhelev for your reply. 
@SergeyKanzhelev, @cijothomas, We are looking to standardize the Tracing, Logging and Metrics using OpenTelemetry for wcf, webapi applications. I am seeing OpenTelemetry with Application Insight as the best bet for my application scenarios for all 3 pillars (Tracing, Logging, and Metrics). Are there any best other exporters for the 3 pillars (Tracing, Logging and Metrics)?

**Comment by SergeyKanzhelev** [comment]
@cijothomas do we need discussions in .NET repo? Or issues would suffice for now?

**Reply by cijothomas** [reply]
Issues and Slack channel should be good for now in my opinion! (1 less place to look for info!)

**Comment by rkbodenner** [comment, marked as answer]
The OTLP exporter supports traces and metrics for .NET now: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol

Logs support is in a separate package at the moment, but should be merged in once we have a stable logs data model. See https://github.com/open-telemetry/opentelemetry-dotnet/pull/2488
